gradle.properties

```diff
@@ -1,8 +1,9 @@
 projectVersion=1.1.2-SNAPSHOT
-micronautVersion=3.3.0
-micronautTestVersion=2.3.7
-groovyVersion=3.0.9
-spockVersion=2.0-groovy-3.0
+micronautVersion=3.8.5
```

Comment:
issue: by upgrading the different libraries we can't make sure that test resources will remain compatible with 3.3+
suggestion: revert the change, test resources 2.0.0 will bump to Micronaut 4 as the baseline

Reply:
done